rim/migrations/0012_auto_20200311_1052.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.0.4 on 2020-03-11 15:52
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('rim', '0011_auto_20200310_1024'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='equipment',
+            name='count',
+            field=models.IntegerField(blank=True, null=True, validators=[django.core.validators.MinValueValidator(0)]),
+        ),
+        migrations.AlterField(
+            model_name='equipment',
+            name='purchase_price',
+            field=models.DecimalField(blank=True, decimal_places=2, max_digits=5, null=True, validators=[django.core.validators.MinValueValidator(0)]),
```

Comment:
This line is inadvertently reverting the changes from #55. Just manually edit this line to be `decimal_places=2, max_digits=7` instead and it should be good to go.

Reply:
Done.